include/dlaf/communication/broadcast_panel.h

```diff
@@ -180,12 +183,14 @@ void broadcast(comm::IndexT_MPI rank_root, matrix::Panel<axis, T, D>& panel,
       panelT.setTile(indexT, panel.read({coord, index_diag_local}));
 
       if (dist.commGridSize().get(comm_coord_step2) > 1)
-        ex::start_detached(scheduleSendBcast(chain_step2(), panelT.read_sender(indexT)));
+        ex::start_detached(scheduleSendBcast(ex::make_unique_any_sender(chain_step2()),
+                                             ex::make_unique_any_sender(panelT.read_sender(indexT))));
     }
     else {
       if (dist.commGridSize().get(comm_coord_step2) > 1)
-        ex::start_detached(
-            scheduleRecvBcast(chain_step2(), owner_diag, panelT.readwrite_sender(indexT)));
+        ex::start_detached(scheduleRecvBcast(ex::make_unique_any_sender(chain_step2()), owner_diag,
+                                             pika::execution::experimental::make_unique_any_sender(
```

Comment:
```suggestion
                                             ex::make_unique_any_sender(
```

probably this needs also a clang-format after the change...